tests/unit-tests/server/class-llms-rest-test-enrollments.php

```diff
@@ -372,7 +372,7 @@ public function test_create_enrollment() {
 		$this->assertEquals( $user_id, $res_data['student_id'] );
 		$this->assertEquals( $course_id, $res_data['post_id'] );
 		$this->assertEquals( 'enrolled', $res_data['status'] );
-		$this->assertEquals( $date_now, $res_data['date_created'], '', $this->date_delta );
+		$this->assertEqualsWithDelta( $date_now, $res_data['date_created'], 10);
```

Comment:
This misses the actual issue.

We're comparing two date strings here so the existing delta (which is 120 and which doesn't need to be reduced) isn't working not because of the function being used but because we're comparing strings not integers.

Furthermore, the current version of phpunit that we're using (due to dependency limitations and support for older versions of PHP) doesn't include this function.

Reply:
I got it, I'm sorry but I'm not having much success with it, the project hardly runs, it's a little complicated, I'm sorry I can't help, thanks for the trust!

Reply:
No problem!